rcgen/src/lib.rs

```diff
@@ -1252,7 +1253,7 @@ impl CertificateParams {
 		pub_key: &K,
 		issuer: &KeyPair,
 		issuer_name: &DistinguishedName,
-	) -> Result<Vec<u8>, Error> {
+	) -> Result<CertificateDer<'static>, Error> {
```

Comment:
Could you also mention in the docs how this can be converted into a vec or byte slice so that it can be saved into a file, ideally with an example that we test or rustdoc links so that we notice the breakage.